### ISSUE TITLE
correct eslint-errors

### ISSUE THREAD

**Issue by adnasa** [vutran/dext#58]
```
Icon.js
  49:11  error  'icon.type' PropType is defined but prop is never used    react/no-unused-prop-types
  50:11  error  'icon.path' PropType is defined but prop is never used    react/no-unused-prop-types
  51:13  error  'icon.letter' PropType is defined but prop is never used  react/no-unused-prop-types

ResultList.js
  54:10  error  'theme' PropType is defined but prop is never used          react/no-unused-prop-types
  55:12  error  'results' PropType is defined but prop is never used        react/no-unused-prop-types
  55:30  error  'results.*' PropType is defined but prop is never used      react/no-unused-prop-types
  56:18  error  'selectedIndex' PropType is defined but prop is never used  react/no-unused-prop-types
```

**Comment by vutran** [comment]
Indirect references causes lint errors. We would just need to reference the props directly.

Issue report here: yannickcr/eslint-plugin-react#833

**Comment by vutran** [comment]
Merged #62
